TS3QueryLib.Core.Framework/Common/Entities/SpacerInfo.cs

```diff
@@ -30,14 +30,14 @@ public static SpacerInfo Parse(string channelName)
             if (channelName.IsNullOrTrimmedEmpty())
                 return null;
 
-            const string pattern = @"\[(?<Alignment>r|l|c|\*)spacer.*?\](?<VisibleName>.*)";
+            const string pattern = @"\[(?<Alignment>r|l|c|\\*)spacer.*?\](?<VisibleName>.*)";
```

Comment:
Why did you add a second backslash? the first one was an escape for the star (*).

Valid values are:
* [*spacer]---
* [rspacer]---
* [cspacer]---
* [lspacer]---

The regex you've made is not correct, because it allows:
* [rspacer]---
* [cspacer]---
* [lspacer]---
* [\spacer]---     (<--Wrong)
* [\\\\\\spacer]---     (<--Wrong)

Regards,
Scordo